### DISCUSSION TITLE
postal-caddy (Restarting)

### DISCUSSION THREAD

**Discussion by SaasX** [postalserver/postal#2489] [Help with using Postal]
Hi all,

I have a problem with docker after a reboot.

Caddy no longer launches :
`c6b15ea3ff84   caddy                               "caddy run --config …"   2 hours ago         Restarting (1) Less than a second ago `

I try to stop, start, reboot again, no way to start caddy. 

An idea ?

**Comment by sinnbeck** [comment]
Maybe try stopping/removing the container and just make it again

```
docker stop c6b15ea3ff84
docker rm c6b15ea3ff84
docker run -d \
   --name postal-caddy \
   --restart always \
   --network host \
   -v /opt/postal/config/Caddyfile:/etc/caddy/Caddyfile \
   -v /opt/postal/caddy-data:/data \
   caddy

**Reply by SaasX** [reply]
Work ! 
Just need to restart before install container again.
Thank you.

**Comment by willpower232** [comment]
I would imagine there was a port conflict resolved by your restarting, you can usually use `docker logs postal-caddy` (or whatever the name of the container is) to see its output and what it was struggling with.
